test/images/tag.svg

```diff
@@ -1,52 +1,52 @@
 <svg width="1000" height="1000" viewBox="0 0 1000 1000"
-	xmlns="http://www.w3.org/2000/svg">
-	<g fill="url(#grad1)" mask="url(#vg)" transform="rotate(0 0 0) translate(0 0) scale(1.0)">
-		<path d="M420.82388 380H588.68467V422.805317H420.82388Z" stroke-width="0" stroke-dasharray="100%" stroke-dashoffset="0" />
-		<path d="m420.82403 440.7101v63.94623l167.86079 25.5782V440.7101Z"/>
-		<path d="M420.82403 523.07258V673.47362L588.68482 612.59701V548.13942Z"/>
-		<g stroke-width="3.864492">
-			<path d="M256.61221 100.51736H752.8963V386.99554H256.61221Z"/>
-			<path d="M201.875 100.51736H238.366478V386.99554H201.875Z"/>
-			<path d="M771.14203 100.51736H807.633508V386.99554H771.14203Z"/>
-		</g>
-	</g>
-	<g fill="url(#grad2)" clip-path="url(#clip)">
-		<path d="M420.82403 691.37851 588.68482 630.5019 589 834H421Z"/>
-		<path d="m420.82403 852.52249h167.86079v28.64782H420.82403v-28.64782 0 0"/>
-		<path d="m439.06977 879.17031c0 0-14.90282 8.49429-18.24574 15.8161-4.3792 9.59153 0 31.63185 0 31.63185h167.86079c0 0 4.3792-22.04032 0-31.63185-3.34292-7.32181-18.24574-15.8161-18.24574-15.8161z"/>
-	</g>
-	<g>
-		<g></g>
-		<polyline fill="none" stroke-width="10" 
+        xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is that svg indentation wrong? Do we really need to change it in this patch?
Or it's just taste/preference problem?

Reply:
I didn't change that, I just added some tags

Reply:
I did change it... sorry, it wasn't my intention

Reply:
it's not a problem, I paste the file from some other editor and it changed it.
fixed
I'm sorry for this